core/base-service/legacy-request-handler.spec.js

```diff
@@ -102,7 +102,7 @@ describe('The request handler', function() {
 
     it('should return the expected response', async function() {
       const { statusCode, body } = await got(`${baseUrl}/testing/123.json`, {
-        json: true,
+        responseType: 'json',
```

Comment:
Could the shorter `.json()` be used instead?

Reply:
Not here: Chaining `.json()` just returns the body, not the whole response. That's why I'm using `.json()` where we're we only want the body e.g:

```js
const body = await got('https://foo.bar').json()
```

but `responseType: 'json'` where we want other data from the response e.g:

```js
const { statusCode, body } = await got('https://foo.bar', {responseType: 'json'})
```

Reply:
Roger that, thanks for clarifying! Just noticed it whilst briefly going through the changelog, thought I would mention it 😄